FirebaseCombineSwift/Sources/Auth/Auth+Combine.swift

```diff
@@ -420,19 +422,19 @@
 
     // MARK: - Other Authentication providers
 
-    #if !os(tvOS) && !os(macOS)
+    #if os(iOS) || targetEnvironment(macCatalyst)
 
       /// Signs in using the provided auth provider instance.
       ///
       /// The publisher will emit events on the **main** thread.
       ///
       /// - Parameters:
       ///   - provider: An instance of an auth provider used to initiate the sign-in flow.
-      ///   - uiDelegate: Optionally an instance of a class conforming to the `AuthUIDelegate`
+      ///   - uiDelegate: Optionally, an instance of a class conforming to the `AuthUIDelegate`
       ///   protocol. This is used for presenting the web context. If `nil`, a default `AuthUIDelegate`
       ///   will be used.
       /// - Returns: A publisher that emits an `AuthDataResult` when the sign-in flow completed
-      ///   successfully, or an error otherwise. The publisher will emit on the *main* thread.
```

Comment:
Is this intended?

Reply:
It wasn't, I was removing duplicate mentions of  _The publisher will emit on the *main* thread._ and happened to miss this. However, Peter brought up below that it's likely better to make that fact overly explicit for developers reading the docs so I'll add the removed statements back.